### ISSUE TITLE
install/update does not support tag

### ISSUE THREAD

**Issue by hangxie** [jsonnet-bundler/jsonnet-bundler#57]
It works fine with branch though, to reproduce the problem:

1. fork https://github.com/anguslees/kustomize-libsonnet
2. create branch `test-branch` and tag `test-tag` without code change, so they are all the same as `master`
3. try to install with different "versions":
* master works fine:
```
jb install git+ssh://git@github.com:hangxie/kustomize-libsonnet.git
...
Branch 'master' set up to track remote branch 'master' from 'origin'.
Already on 'master'
CLEAN vendor/terraform-jsonnet
```
* branch works fine:
```
jb install git+ssh://git@github.com:hangxie/kustomize-libsonnet.git@test-branch
...
Branch 'test-branch' set up to track remote branch 'test-branch' from 'origin'.
Switched to a new branch 'test-branch'
```
* tag does not work:
```
jb install git+ssh://git@github.com:hangxie/kustomize-libsonnet.git@test-tag
...
From github.com:hangxie/kustomize-libsonnet
 * tag               test-tag   -> FETCH_HEAD
error: pathspec 'test-tag' did not match any file(s) known to git
jb: error: failed to install packages: downloading: exit status 1
```

Just in case it matters:
```
$ git version
git version 2.23.0
```

I went through source code quickly, and found https://github.com/jsonnet-bundler/jsonnet-bundler/blob/master/pkg/git.go#L244, `git fetch --depth 1 origin <version>` handles branch only, and I need to use `git fetch --depth 1 origin --tags <version>` to handle tag. The `--tags`/`--no-tags` behavior is part of git-config so I believe it needs to use the explicit value here (https://git-scm.com/docs/git-fetch, search for `tagOpt`).

**Comment by davidovich** [comment]
I believe this can be closed by #58 being merged.

**Comment by hangxie** [comment]
Would you please cut a release like v0.1.1? We may heavily rely on jb so prefer to have a stable version to use.
